.travis.yml

```diff
@@ -7,6 +7,6 @@ install:
 script:
   - "pep8 ."
   - "pylint --load-plugins pylint_django volontulo"
-  - "py.test -v --cov . --cov-report term-missing"
+  - "py.test --ds=volontulo.pytest_settings -v --cov . --cov-report term-missing"
```

Comment:
-ds has been enabled by pytest-django - it points to Django settings modulo.

http://pytest-django.readthedocs.org/en/latest/configuring_django.html#command-line-option-ds-settings

BTW this branch works properly on my dev environment but it fails on Travis, so I need to figure out why and we will merge it then.